### PR TITLE
3 layers! move ProposePlaceService

### PR DESCRIPTION
**Pull request by olenapetryshak**
https://github.com/ita-social-projects/GreenCity/issues/1614

### CODE REVIEW

File: service-api/pom.xml [hunk 1]
```diff
@@ -37,6 +40,16 @@
             <groupId>org.hibernate.validator</groupId>
             <artifactId>hibernate-validator</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
```
Comment (antonbakalets) [inline service-api/pom.xml:45]: Those two should not be here.

File: service-api/src/main/java/greencity/dto/breaktime/BreakTimeDto.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import lombok.*;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@EqualsAndHashCode(exclude = {"startTime", "endTime"})
+public class BreakTimeDto {
+    @NotNull
+    @JsonFormat(pattern = "HH:mm", shape = JsonFormat.Shape.STRING)
```
Comment (antonbakalets) [inline service-api/src/main/java/greencity/dto/breaktime/BreakTimeDto.java:16]: Those two annotation should be on the bean accepted by controller.

File: service-api/src/main/java/greencity/dto/openinghours/OpenHoursDto.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import java.time.DayOfWeek;
+import java.time.LocalTime;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+public class OpenHoursDto implements Comparable<OpenHoursDto> {
```
Comment (antonbakalets) [inline service-api/src/main/java/greencity/dto/openinghours/OpenHoursDto.java:14]: Is this class used somewhere?

File: service-api/pom.xml (hunk 1 above)
Comment (olenapetryshak) [reply service-api/pom.xml]: Ok. Thanks!

File: service-api/src/main/java/greencity/dto/breaktime/BreakTimeDto.java (hunk 2 above)
Comment (olenapetryshak) [reply service-api/src/main/java/greencity/dto/breaktime/BreakTimeDto.java]: removed

File: service-api/src/main/java/greencity/dto/openinghours/OpenHoursDto.java (hunk 3 above)
Comment (olenapetryshak) [reply service-api/src/main/java/greencity/dto/openinghours/OpenHoursDto.java]: removed. it used in PlaceService. I will move it in my next task.

**Review by Yura09 — APPROVED** [review]

**Review by eyouu — APPROVED** [review]

**Review by IvanBegar — APPROVED** [review]

**Review by antonbakalets — APPROVED** [review]

**Pull request merged**